test/Generator/PropertyGeneratorTest.php

```diff
@@ -281,4 +281,12 @@ public function testSetDefaultValue(string $type, $value) : void
         self::assertEquals($type, $property->getDefaultValue()->getType());
         self::assertEquals($value, $property->getDefaultValue()->getValue());
     }
+
+    public function testOmitType()
+    {
+        $property = new PropertyGenerator('test', null);
+        $property->setDefaultValue(null, ValueGenerator::TYPE_OMIT);
```

Comment:
That's really really ugly. Would rather prefer `->removeDefaultValue()` or `->omitDefaultValue()` instead of having a `ValueGenerator` with special purposes (there is a big impedance between the concept of a value generator and the concept of a lack of value)

Reply:
I'll remove the constant, and move it to a property in the PropertyGenerator.

Reply:
If you look at the other comments, I suggest not touching the `PropertyGenerator` at all